These pytest plugins are not needed to run tests.
We do not have them packaged (yet).

--- cheroot-10.0.1/requirements/tests.in.orig
+++ cheroot-10.0.1/requirements/tests.in
@@ -27,13 +27,10 @@
 pyopenssl >= 22.0.0; implementation_name == "pypy" and python_version >= "3.8"
 pyopenssl < 22.0.0; implementation_name == "pypy" and python_version < "3.8"
 
-pytest-clarity
 pytest-cov==2.12.0
 pytest-forked>=1.2.0; sys_platform != "win32"
 pytest-mock>=1.11.0
 pytest-rerunfailures
-pytest-sugar>=0.9.3
-pytest-watch==4.2.0
 pytest-xdist>=1.28.0
 
 # pytest-forked is currently incompatible with pytest 7
